src/test/regress/expected/coordinator_shouldhaveshards.out

```diff
@@ -908,7 +908,7 @@ key
 FROM a JOIN table_2 USING (key)
 GROUP BY key
 HAVING (max(table_2.value) >= (SELECT value FROM a));
-DEBUG:  Group by list without distribution column is not allowed  in distributed INSERT ... SELECT queries
+DEBUG:  complex joins are only supported when all distributed tables are co-located and joined on their distribution columns
```

Comment:
not sure if this is the correct reason

Reply:
We use `a` two times in the query. I guess ` a` on this one `SELECT value FROM a` is not colocated with the rest of the tables, so probably this is also a right error message.

I guess the order of the error checks have changed